modules/site-health/admin-site-health.php

```diff
@@ -0,0 +1,188 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+} // Exit if accessed directly.
+
+/**
+ * Class PLL_Admin_Site_Health to add debug info in WP Site Health
+ *
+ * @since 2.8
+ * @link https://make.wordpress.org/core/2019/04/25/site-health-check-in-5-2/
+ */
+class PLL_Admin_Site_Health {
+	/**
+	 * PLL_Admin_Site_Health constructor.
+	 *
+	 * @since 2.8
+	 */
+	public function __construct() {
+		// Information tab
+		add_filter( 'debug_information', array( $this, 'pll_info_options' ), 15 );
+		add_filter( 'debug_information', array( $this, 'pll_info_languages' ), 16 );
+
+		// tests Tab
+		add_filter( 'site_status_tests', array( $this, 'pll_is_homepage' ) );
+	}
+
+	/**
+	 * Add Polylang Options to Site Health Informations tab.
+	 *
+	 * @param array $debug_info array options to display.
+	 * @return array
+	 * @since   2.8
+	 */
+	public function pll_info_options( $debug_info ) {
+		$options = get_option( 'polylang' );
+		$fields = array();
+		foreach ( $options as $key => $value ) {
+			if ( ! is_array( $value ) ) {
+				switch ( $key ) {
+					case 'first_activation':
+						$fields[ $key ]['label']   = $key;
+						$fields[ $key ]['value']   = date_i18n( 'd/m/Y', $value );
+						$fields[ $key ]['private'] = false;
+						break;
+					default:
+						$fields[ $key ]['label']   = $key;
+						$fields[ $key ]['value']   = $value;
+						$fields[ $key ]['private'] = false;
+						break;
```

Comment:
```php
$fields[ $key ] = array(
	'label'   => $key,
	'value'   => $value,
	'private' => false,
);
```
would be more readable to me. Let say that's in our coding standards although wie have no WPCS sniff to check this ;-).

Reading the post you linked to, it looks like the `private` key is not necessary when to false.